### DISCUSSION TITLE
Detected by Sony and ArkoseLabs?

### DISCUSSION THREAD

**Discussion by FlowFlaux** [seleniumbase/SeleniumBase#2561] [Q&A] [answered]
I've been trying to use SB to log into my PSN account though store.playstation.com, and I keep getting timed out by the login page, even if I solve the FunCaptcha correctly.
I am using Chrome in UC mode, and I am using uc_click method to click the 'Sign In' buttons once the fields are filled. I've also tried using selenium_stealth, alternating useragents and also Opera and Brave. 
Is there anything else I might be missing?

**Comment by mdmintz** [comment, marked as answer]
Unless regular human browser traffic is also required to solve those CAPTCHAs, the Selenium browser shouldn't have to solve those CAPTCHAs either (because otherwise it has already been detected, and solving the CAPTCHA won't help).

Your issue might be timing related. Try getting through from the special UC breakpoint:

```python
sb.driver.uc_open_with_reconnect(URL, "breakpoint")
```

If you can get through from there manually, then you can probably get through using the existing UC Mode methods and adjusting for timing.

**Comment by FlowFlaux** [comment]
So I've done some step by step tests, and it looks like it's getting detected when it starts inputting text.
I've tried to use the .type method, as well as send_keys and press_keys, but all three seem to be getting detected.
Is there a better way to do that?

**Reply by mdmintz** [reply]
If they're catching you *while* selenium is typing, that's a tricky one to avoid. Maybe an `sb.js_type(selector, text)`, followed immediately by a `sb.driver.reconnect(2)`, but no guarantees.

**Reply by FlowFlaux** [reply]
js_type doesn't seem to be filling the forms at all, although it allows the buttons to be pressed and results in an error.
The detection is weird. Just the 'type' method seems to work sometimes, and then it stops.
What's clear is that it's the typing and form submission that are causing detection. Just opening the page, disconnecting the driver and filling the forms out manually works without issue.

**Reply by mdmintz** [reply]
There's an example for when you have to bypass a CAPTCHA on a form: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

**Reply by FlowFlaux** [reply]
Is there a way to disconnect the driver completely, so I could try entering the details with something like pyautogui?

**Reply by mdmintz** [reply]
You could try:

```python
sb.driver.uc_open_with_reconnect(URL, reconnect_time="breakpoint")
sb.driver.uc_open_with_reconnect(URL, "breakpoint")
sb.driver.reconnect(timeout="breakpoint")
sb.driver.reconnect("breakpoint")
```

**Reply by FlowFlaux** [reply]
I just hacksawed the reconnect function in half and turned it into disconnect and connect ones x)
Thank you for your insights! It's an awesome tool and we are lucky to have you maintaining it so actively.
